index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="kelwilson" />
+    <title>kelwilson's portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="portfolio-wrap">
+      <div class="background-icons">
+        <div class="background- img">
+          <img
+            src="images/image_geometry_1.png"
+            class="background-image"
+            alt=""
+          />
+        </div>
+
+        <div class="toggle">
+          <div class="navigation">
+            <img src="images/ic_menu@2x.png" class="menu-btn" alt="" />
+          </div>
+        </div>
+        </div>
```

Comment:
- [ ] We often describe the header as anything on top of the website, and it is mostly used to host the navigation features of the website. Kindly change this block to be wrapped by a `header` tag